jkube-kit/generator/api/src/main/java/org/eclipse/jkube/generator/api/support/BaseGenerator.java

```diff
@@ -227,6 +231,14 @@ protected void addLatestTagIfSnapshot(BuildConfiguration.BuildConfigurationBuild
         }
     }
 
+    protected void addTagsFromConfig(BuildConfiguration.BuildConfigurationBuilder buildConfigurationBuilder) {
+        String commaSeparatedTags = getConfig(Config.TAGS);
+        if (StringUtils.isNotBlank(commaSeparatedTags)) {
+            String[] tags = commaSeparatedTags.trim().split(",");
```

Comment:
Each tag should be trimmed, consider the following config:
`<generator.tags> tag-1,    tag-2    , other-tag</>`